syntax = "proto2";
import "ignition/msgs/time.proto";
package smartmouse.msgs;

message WorldStatistics {
    optional uint64 steps = 1;
    optional ignition.msgs.Time sim_time = 2;
}
